src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-infra-hosts.mdx

```diff
@@ -0,0 +1,308 @@
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `processors::resourcedetection`
+      </td>
+      <td>
+        Keep the following in mind:
+
+          * `env`: Reads resource information from the `OTEL_RESOURCE_ATTRIBUTES` environment variable.
+          * `system`: Adds `host.name` and `os.type`.
+          * For cloud environments, a specific resource detection processor needs to be configured so that metrics are decorated with `host.id` (required to identify the Host entity in New Relic One). Common cloud detectors are `gce` for GCP machines, `ec2` for AWS EC2 and `azure` for Azure VMs. Additional processors are available for orchestrated environments.
+          * For on-premises systems (or unsupported cloud environments), a `host.id` attribute is required. The `resource` processor can be used to copy the `host.name` value (from `system`) as a new `host.id` attribute. Note this value should be unique across all instrumented hosts:
+          ```
+          resource:
+              attributes:
+                - key: host.id
+                  from_attribute: host.name
+                  action: upsert
+          ```
+
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `processors::batch`
+      </td>
+      <td>
+        The [batch processor](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/batchprocessor/README.md) accepts spans, metrics, or logs and places them into batches. Batching helps better compress the data and reduce the number of outgoing connections required to transmit the data. This processor supports both size and time based batching.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `processors::memory_limited`
+      </td>
+      <td>
+        The [memory limiter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/memorylimiterprocessor) processor is used to prevent out of memory situations on the collector. Given that the amount and type of data the collector processes is environment specific and resource utilization of the collector is also dependent on the configured processors, it is important to put checks in place regarding memory usage.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `processors::cumulative_delta`
+      </td>
+      <td>
+        The [cumulative delta](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/cumulativetodeltaprocessor) processor converts cumulative sum metrics to cumulative delta which is recommended in order to query system metric rates more easily in New Relic One.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `service::pipelines::metrics`
+      </td>
+      <td>
+        Make sure `hostreceiver` and `resourcedetection` are included.
+      </td>
+    </tr>
+    </tbody>
+</table>
+
+Here is a sample configuration YAML file:
```

Comment:
I'd add a note: Replace NEW_RELIC_OTLP_ENDPOINT and INGEST_LICENSE_KEY with... (same content from the basic setup)